docs/business-apps/power-apps/get-started/create-your-first-custom-form.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Customize a form for a SharePoint list
+description: Customize a form for a SharePoint list
+ms.date: 06/03/2020
+ms.prod: sharepoint
+localization_priority: Priority
+---
+
+# Customize a form for a SharePoint list
+
+Using Power Apps, you can easily customize a form for a SharePoint list that works best for your team or organization.
+
+In this tutorial, you will create a custom form with a user profile photo, a read-only field, and a conditionally-visible field.
+
+## Required setup
+
+Before starting, make sure your SharePoint site is set up with the [required lists and libraries](https://docs.microsoft.com/sharepoint/dev/business-apps/get-started/set-up-sharepoint-site-lists-libraries).
+
+## Create a custom form
+
+1. In your SharePoint site, browse to the **Projects** list.
+
+2. To create and manage flows for a list or a library, from the command bar, on the list or library page, select **Power Apps**. More options appear:
+
+   - Create an app
+   - See all apps
+   - Customize forms
+
+    ![Create a custom form from Projects list](../../../images/gs01-create-a-custom-form-command-bar.png)
+
+3. Select **Customize forms**. The Power Apps studio appears and loads your form onto the canvas. If the **Welcome to Power Apps Studio** dialog box opens, select **Skip**.
+
+    ![Power Apps studio with custom form](../../../images/gs01-power-apps-studio-custom-form.png)
+
+### Add a user profile photo
+
+1. On the **Insert** tab, select **Media** > **Image**. An [Image](https://docs.microsoft.com/en-us/powerapps/maker/canvas-apps/controls/control-image) control named **Image1** is added to the canvas and to the [Tree view](https://docs.microsoft.com/en-us/powerapps/maker/model-driven-apps/using-tree-view-on-form) on the left-hand panel.
```

Comment:
Don't include locales in URLs... the site uses the user's browser agent to redirect them to localized pages. This is present in multiple places... I'm only going to add the comment here but applies to entire PR.